### DISCUSSION TITLE
Is it possible: chatwoot + apartment gem

### DISCUSSION THREAD

**Discussion by stap780** [chatwoot/chatwoot#2574] [Self hosted]
Hello,
is it possible to use chatwoot self-hosted with apartment gem?

or for every use we need setup new server?

**Comment by sojan-official** [comment]
You can have multiple accounts in a single chatwoot installation. 
So I don't think you would need to use apartment gem.

**Reply by stap780** [reply]
Thanks for answered. Can you give me link to the documentation where i can read about it? Don't find by myself.

**Reply by stap780** [reply]
To be clear. I am talk about next. For example.
We have one server with chatwoot. Next we have client1 and client2. 
Client1 (is one company) have several users. 
Client2 (is another company ) have several users. 
Client1 users log in to chatwoot and work only with their customers requests.
Client2 users log in to chatwoot and work only with their customers requests.
and etc.

I am write about apartment because it can give simple imagination about what i am talk.
We maked a little saas service (k-comment.ru) for Insales clients and use apartment gem to have multitenant and separate database for our users.

**Reply by sojan-official** [reply]
You should be able to provision additional accounts into the installation using either super admin or using the platform APIs

ref :
https://www.chatwoot.com/docs/self-hosted/monitoring/super-admin-sidekiq/
https://github.com/chatwoot/chatwoot/wiki/Building-on-Top-of-Chatwoot:-Platform-APIs 
https://www.chatwoot.com/developers/api#tag/Accounts

**Reply by stap780** [reply]
Thanks very much for reply.

One last question :)

Can you advise me what kind of setup use because we want to have last updates in future.
And how better integrate telegram that we can make base chatwoot updates later.
